CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Unreleased
+* Upgraded Octokit to 5.6.1 (#1327)
+* Migrate from legacy Rails secrets to credentials (#1326)
+  * Rails secrets were [deprecated in Rails 7.1](https://github.com/rails/rails/pull/48472)
+  * [Guide on credentials](https://guides.rubyonrails.org/security.html#custom-credentials)
+  * Migrated using the ejson-rails gem and [these migration instructions](https://github.com/Shopify/ejson-rails?tab=readme-ov-file#migrating-to-credentials)
```

Comment:
`ejson-rails` only concerns Shopify. It's a public gem so people can use it if they way, but as far as `shipit-engine` (the open source project) is concerned, it only migrated to Rails credentials, not to `ejson-rails`.